api/app/GraphQL/Mutations/QualifyCandidate.php

```diff
@@ -24,6 +24,10 @@ public function __invoke($_, array $args)
         $candidate->final_decision_at = $now;
         $candidate->save();
 
+        $finalDecicion = $candidate->computeFinalDecision();
+        $candidate->computed_final_decision = $finalDecicion['decision'];
+        $candidate->computed_final_decision_weight = $finalDecicion['weight'];
+
         return $candidate;
```

Comment:
Need to move `candidate->save();` after the new changes to candidate.

Other files this needs to fixed in includes:

- [ ] DisqualifyCandidate.php

Reply:
Missed QualifyCandidate :sweat_smile: then should be good :+1:

Reply:
Fixed in [fix save order when qualifying candidate](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10920/commits/070671adf3233accbf65abef486fd1d23a70f240)